mailboxes/lang/de

```diff
@@ -380,58 +383,58 @@ acl_fdoms=Mailbox @ Domains
-razor_report3=Entferne die gewählten Nachrichten aus Razor und andere SpamAssassin-Spamfilter-Datenbanken ..
-razor_done=.. erledigt
-razor_err=.. gescheitert! Oben sehen Sie warum.
-razor_moved=.. erledigt, und nach Ordner $1 verschoben.
-razor_deleted=.. erledigt. Die Mail wurde gelöscht.
+razor_title2=Als kein Spam melden
+razor_report=Diese Nachricht wird sowohl der Razor- als auch anderen, durch SpamAssassin genutzten, Anti-Spam-Datenbanken gemeldet
+razor_report2=Die gewählten Nachrichten an Razor und andere SpamAssassin-Spamfilter-Datenbanken melden
+razor_report3=Die gewählten Nachrichten aus Razor und andere SpamAssassin-Spamfilter-Datenbanken entfernen
+razor_done=Erledigt.
+razor_err=Fehlgeschlagen!
+razor_moved=Erledigt und nach Ordner $1 verschoben.
+razor_deleted=Erledigt: Die Mail wurde gelöscht.
 
 ham_title=Als Nicht-Spam melden
-ham_report=Diese Nachricht wird als Nicht-Spam an Razor und andere SpamAssassin-Datenbanken gemeldet ..
+ham_report=Diese Nachricht wird als Nicht-Spam an Razor und andere SpamAssassin-Datenbanken gemeldet
 
 black_title=Absender verbieten
-black_done=Die Mail-Adresse wurde der Liste der durch SpamAssassin abgewiesenen Adressen hinzugefügt.
+black_done=Die Mail-Adresse wurde der Liste der durch SpamAssassin abgewiesenen Adressen hinzugefügt
 black_already=Die Mail-Adresse $1 befindet sich bereits auf der Liste der durch SpamAssassin abgewiesenen Adressen
 
-white_title=Erlaube Absender
-white_done=Die Mail-Adresse $1 wurde der Whitelist von SpamAssassin hinzugefügt.
-white_already=Die Mail-Adresse $1 befindet sich bereits in der Whitelist von SpamAssassin.
+white_title=Absender erlauben
+white_done=Die Mail-Adresse $1 wurde der SpamAssassin-Whitelist hinzugefügt
+white_already=Die Mail-Adresse $1 befindet sich bereits in der SpamAssassin-Whitelist
 
-ldap_emod=Das fehlende Perl-Modul $1 wird benötigt, um sich via LDAP zu verbinden
+ldap_emod=Das fehlende Perl-Modul $1 wird benötigt, um eine Verbindung via LDAP herzustellen
 ldap_econn=Fehler beim Verbinden zum LDAP-Server $1 Port $2
 ldap_elogin=Konnte nicht zum LDAP-Server $1 als $2 verbinden : $3
-ldap_ehost=LDAP Server in der Modulkonfiguration nicht gesetzt
-ldap_eport=Kein gültiger LDAP Server Port in der Modulkonfiguration gesetzt
-ldap_euser=Kein LDAP Login in der Modulkonfiguration gesetzt
-ldap_ebase=LDAP base DN in Modulkonfiguration nicht gesetzt
+ldap_ehost=LDAP-Server in der Modulkonfiguration nicht gesetzt
+ldap_eport=Kein gültiger LDAP-Server-Port in der Modulkonfiguration gesetzt
+ldap_euser=Kein LDAP-Login in der Modulkonfiguration gesetzt
+ldap_ebase=LDAP-Base-DN in Modulkonfiguration nicht gesetzt
 
 delall_title=Alle Mails löschen
-delall_rusure=Sind Sie sicher, dass Sie alle Mails von $1 löschen wollen? $2 von $3 Nachrichten werden unwiederbringlich gelöscht.
+delall_rusure=Sollen alle Mails von $1 gelöscht werden? $2 von $3 Nachrichten werden unwiderruflich gelöscht
```

Comment:
Dots at the end of a phrase should be preserved generally, in most cases. However, since this is more of a question in a phrase rather than in a label, you should use a `?` mark here.